### PR TITLE
Buffs helmets melee armor

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Buffs helmets melee armor from 50 to 65

## Why It's Good For The Game

Head already has less health and worse injuries than chest,having the same armor as the chest piece makes no sense,this should fix the head targeting meme

## Changelog
:cl:
balance:marine helmets melee armor increased
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by David-Stormwell** [comment]
gey

**Comment by Kuro-ai** [comment]
> gey

noob

**Comment by TiviPlus** [comment]
We dont need more marine armor buffs right now

**Comment by SplinterGP** [comment]
Tivi????

**Comment by SplinterGP** [comment]
This was an oversight last time we touched armor, like handa and feet head has less health and worst fractures, not to mention it should NOT have less armor than a chestpiece, what would be the point then? Oh ye let me take a tyr to have more slowdown and armor then get targetted head and die as if i had no tyr.

**Review by SplinterGP — APPROVED** [review]

**Comment by Swagile** [comment]
this is retarded and the fact that a maintainer actually approved this is even more so.

please play the game splinter

**Comment by Swagile** [comment]
ok let me put forth more information so i don't sound like an asshole (even though i am, but im not a dumb asshole)

there are currently only 3 to 5 people who start with roundstart 75 melee armor helmets (FOR GOOD REASON)

2 SL's (PoS) or 4 SL's (Sulaco), and FC helmet

every other roundstart helmet starts at 50

now lets put this in perspective;

B17 armor is 60 points for ONE set

B18 armor is 100 points for ONE set

the helmet armor for both of these is 65.

65 melee armor. for something that requires req.

you now want to buff every helmet's melee resistance to 75 for absolutely no good reason (tyr 2 is not a good reason, wtf??????????)

tl;dr even without playing the game and as a maintainer of the code base you should know this is a bad idea, doubly so if you played the game

but im talking to a brick wall because even in other PR's ive commented on, shit got merged anyway

whatever

**Comment by TiviPlus** [comment]
What swagile said basically

**Comment by SplinterGP** [comment]
> What swagile said basically

what swagile said has no sense

**Comment by TiviPlus** [comment]
Wrong

**Comment by TiviPlus** [comment]
Stop reopening it until there is an agreement

**Comment by TiviPlus** [comment]
Ok fine we do it my way

**Comment by TiviPlus** [comment]
@SplinterGP stop closing until there's an agreement in discord

**Comment by SplinterGP** [comment]
> @SplinterGP stop opening until there's an agreement in discord

stop closing until there is an agreement on discord, you close a thing thats under discussion, the thing is to leave it up

**Comment by TiviPlus** [comment]
@SplinterGP then wait until the discussion is over people keep telling you buffing helmets to this extreme is dumb listen to them please

**Comment by SplinterGP** [comment]
why are you scared of letting other people comment on the balance thing

**Comment by TiviPlus** [comment]
?????

**Comment by TiviPlus** [comment]
Happy yet

**Comment by SplinterGP** [comment]
why cant you just let it open, all balance PR's are like this, i don't see you closing all the others you dont agree with

**Comment by TiviPlus** [comment]
It's a bad Pr you know it I know it everyone keeps telling you it is so just let it die

**Comment by SplinterGP** [comment]
> It's a bad Pr you know it I know it everyone keeps telling you it is so just let it die

literally only swagile did

**Comment by David-Stormwell** [comment]
No, I have a feeling quite a few more people would disagree on this if they knew of it.
Making helmets omega protected is way too much of a power creep, and how it is right now feels fine.

**Comment by MiniMeatwad** [comment]
This is bad.

**Comment by SplinterGP** [comment]
> No, I have a feeling quite a few more people would disagree on this if they knew of it.
> Making helmets omega protected is way too much of a power creep, and how it is right now feels fine.
> And to add to it, Xenos almost never target the head, and head damage is actually pretty easy to treat due to imialky existing.

every limb of game has this, they have more armor than the chest since they have less health and break easier and generally have a upside in exchange of not giving as much damage, such as hand that makes you drop your gun, or if fully removed cant use it, same for arm also makes you drop your gun.
Head also has less health like them and causes brain damage and eye damage which is way worse than a normal fracture on chest.
all targets on the health doll are sidegrades to targetting chest, you shouldt have just clear advantage over another player just because you know a oversight exists or that you know to press 8 and the other player doenst

**Comment by Swagile** [comment]
ok so after talking about it in discord

the argument, from my understanding, is:

Chest has 200 total HP, and a 60 HP break threshold, while head and all other limbs are at a 40 break threshold 100 total HP.

All other limbs EXCEPT head apparently have 75 armor to compensate for their lower health, while head has stayed at 50 armor.

So this PR is supposed to make it so this oversight is fixed because otherwise xenos will just aim at head.

At least thats how I have been told and understand from what ive been told.

Is this correct @SplinterGP ?

**Comment by SplinterGP** [comment]
things being fixed to intended behaviour aint bad, we nerfed marine armor finely when we noticed them having +5 armor than intended because MJP dindt knew codex only showed that piece of armor armor, instead of showing your total armor, same in here, we noticed we forgot to increase helmet armor and we're just fixing it to intended beahviour.

**Comment by SplinterGP** [comment]
> ok so after talking about it in discord
> 
> the argument, from my understanding, is:
> 
> Chest has 200 total HP, and a 60 HP break threshold, while head and all other limbs are at a 40 break threshold 100 total HP.
> 
> All other limbs EXCEPT head apparently have 75 armor to compensate for their lower health, while head has stayed at 50 armor.
> 
> So this PR is supposed to make it so this oversight is fixed because otherwise xenos will just aim at head.
> 
> At least thats how I have been told and understand from what ive been told.
> 
> Is this correct @SplinterGP ?

arm and legs have a 50 break treshold, hand and feet have 37

**Comment by Swagile** [comment]
> 
> 
> > ok so after talking about it in discord
> > the argument, from my understanding, is:
> > Chest has 200 total HP, and a 60 HP break threshold, while head and all other limbs are at a 40 break threshold 100 total HP.
> > All other limbs EXCEPT head apparently have 75 armor to compensate for their lower health, while head has stayed at 50 armor.
> > So this PR is supposed to make it so this oversight is fixed because otherwise xenos will just aim at head.
> > At least thats how I have been told and understand from what ive been told.
> > Is this correct @SplinterGP ?
> 
> arm and legs have a 50 break treshold, hand and feet have 37

And both still have 75 armor, so this is just to fix that, right?

Cause if that is the case, it is pretty reasonable to buff helmets up (though I personally dislike it cause it makes it even fucking harder to deal with current marine meta and their tankiness), but I get why it would be done.

**Comment by Pariah919** [comment]
helmets should probably work like gloves and boots
But that buffs berets so uh, i guess this is fine at 60

**Comment by Kuro-ai** [comment]
I reduced the armor buff to 60,seems more reasonable,I forgot B18 had 70 melee armor helmets

**Comment by SplinterGP** [comment]
> helmets should probably work like gloves and boots
> But that buffs berets so uh, i guess this is fine at 60

this doenst buff berets nerd

**Comment by SplinterGP** [comment]
> I reduced the armor buff to 60,seems more reasonable,I forgot B18 had 70 melee armor helmets

you could give it 65, so it wont just be better than the chest if the player takes a tyr in example, allowing xeno to bypass it.

**Comment by Kuro-ai** [comment]
> > I reduced the armor buff to 60,seems more reasonable,I forgot B18 had 70 melee armor helmets
> 
> you could give it 65, so it wont just be better than the chest if the player takes a tyr in example, allowing xeno to bypass it.

AGONY

**Comment by TiviPlus** [comment]
No

**Comment by Chaznoodles** [comment]
Remember that this also stacks with the PMC damaged masks you can find on certain maps, that also have 5 armour.

**Comment by SplinterGP** [comment]
> Remember that this stacks with the PMC damaged masks you can find on certain maps, that also have 5 armour.

i honestly think those PMC stuff on map is yikes, i think we should work on replacing it with armorless or incase of chestpiece same armor version, like a damaged PMC armor

**Comment by Kuro-ai** [comment]
I have become death,destroyer of PRs

**Comment by Losenis** [comment]
Fucking hell I come back from getting head fracced to hell and back despite running tyr2 and this pr got closed
